pages/api/patients/delete.ts

```diff
@@ -0,0 +1,26 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      let { id } = req.body.data
+
+      var patient = await prisma.patient.delete({
```

Comment:
avoid the use of `var` 
```suggestion
      const patient = await prisma.patient.delete({
```